packages/viz/src/lib/Layer.ts

```diff
@@ -75,6 +125,10 @@ export class Layer {
     return { urlTemplates, geometryType };
   }
 
+  private generateLayerId(layerSource: string) {
+    return `MVTLayer-${layerSource}`;
```

Comment:
How would this work with a long SQL complex query...? Should we create some  hash?